tests/qos/test_qos_dscp_mapping.py

```diff
@@ -280,9 +278,12 @@ def _run_test(self,
             if packet_egressed_success:
                 dut_egress_port = get_dut_pair_port_from_ptf_port(duthost, tbinfo, dst_ptf_port_id)
                 pytest_assert(dut_egress_port, "No egress port on DUT found for ptf port {}".format(dst_ptf_port_id))
+                # Wait for the queue counters polling.
```

Comment:
@vivekverma-arista should use wait until here?

Reply:
Thanks for the suggestion, updated the code to use `wait_until`.